tests/scripts/all.sh

```diff
@@ -2675,10 +2703,18 @@ component_test_psa_crypto_config_accel_rsa_signature () {
     not grep mbedtls_rsa_rsassa_pkcs1_v15_sign library/rsa.o
     not grep mbedtls_rsa_rsassa_pss_sign_ext library/rsa.o
 
-    msg "test: MBEDTLS_PSA_CRYPTO_CONFIG with accelerated RSA signature"
+    if [ $USE_NEW_SYMBOLS == "YES" ]; then
+        msg "test: MBEDTLS_PSA_CRYPTO_CONFIG with accelerated RSA signature (new symbols)"
```

Comment:
Part of the goal of this test is to check that RSA support is present. We're not doing this.